### PR TITLE
IssueID:1701:Workaround for CH395 ethernet chip

### PR DESCRIPTION
**Pull request by wuxn9999**
[Detail]
The CH395 ethernet chip on board is unstable for some reason.
Workaround for stable ethernet connection.

[Verified Cases]
Build Pass: <helloworld_demo@haas100>
Test Pass: <helloworld_demo@haas100>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
start build

**Comment by skylarCai** [comment]
手动触发！
感谢 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
手动触发！
感谢 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : dd04d3c8cec1604a8972745b9586b24976bc7ab0

**Comment by skylarCai** [comment]
auto build success,number:209

**Comment by skylarCai** [comment]
autotest fail,number:1560

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

File: hardware/board/haas100/drivers/ch395_lwip.c [hunk 1]
```diff
@@ -15,6 +15,7 @@
 #include "uservice/uservice.h"
```
Comment (skylarCai) [inline hardware/board/haas100/drivers/ch395_lwip.c:15]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**